docs/customization_guide/inference_protocols.md

```diff
@@ -149,7 +149,7 @@ protocol types mentioned above:
 request to the protocol is received. The completed header will be in the form of
 `triton-grpc-protocol-<restricted-key>`
 
-* `restricted-value` : The value of the header to be matched in order to preceed
+* `restricted-value` : The value of the header to be matched in order to proceed in
```

Comment:
I am not sure I understand both versions old and new.

Reply:
I was only correcting the spelling, @GuanLuo might be able to clarify what this means.